draft-ietf-httpbis-cache-latest.xml

```diff
@@ -1486,27 +1486,25 @@
 <section title="must-revalidate" anchor="cache-response-directive.must-revalidate">
    <iref item="must-revalidate (cache directive)" primary="true" />
 <t>
-   The "must-revalidate" response directive indicates that once it has become
-   stale, the response &MUST-NOT; be used to satisfy any other request
-   without forwarding it for validation and receiving a successful response;
-   see <xref target="validation.model"/>.
+   A "must-revalidate" response directive indicates that once the response
+   has become stale, a cache &MUST-NOT; reuse that response to satisfy another
+   request until it has been successfully validated by the origin, as defined
+   by <xref target="validation.model"/>. Furthermore, a cache &MAY; reuse a
+   response containing must-revalidate even if it is a response to a request
+   containing an Authorization field, subject to the above requirement on
+   revalidation.
 </t>
 <t>
    The must-revalidate directive is necessary to support reliable operation
-   for certain protocol features. In all circumstances a cache &MUST; obey
-   the must-revalidate directive; in particular, if a cache is disconnected,
-   it &MUST; generate a <x:ref>504 (Gateway Timeout)</x:ref> response.
```

Comment:
It's still there, so this must be just a weird diff effect.

Reply:
I'm looking at https://github.com/httpwg/http-core/blob/fielding-i161/draft-ietf-httpbis-cache-latest.xml#L1490 and don't see it

Reply:
Sorry, I see now that I removed the defined term ... will change that part back. I still want to replace the ambiguous "it" (not appropriate for a requirement) and end the sentence with "rather than reuse the stale response".